### ISSUE TITLE
Compatibility with Django 4.1

### ISSUE THREAD

**Issue by niccolomineo** [dabapps/django-enforce-host#8]
As I upgrade to Django 4.1, this error shows up upon requesting a simple favicon:

```
'EnforceHostMiddleware' object has no attribute '_is_coroutine'
```

**Comment by j4mie** [comment]
Thanks for the report @niccolomineo, I'll look into this as soon as I can.

**Comment by j4mie** [comment]
@niccolomineo this is now fixed in version 1.1.0 👍🏻 thanks again!

**Comment by niccolomineo** [comment]
Thanks, works a charm.
